test/rules/experiments/nnbd/rules/unnecessary_null_checks.dart

```diff
@@ -54,3 +54,14 @@ f5(int? p) {
   int? v1;
   v1 ??= p!; // OK
 }
+
+class B {
+  m3a(int? p) => p!; // OK
```

Comment:
Are these cases different from methods in `class A`?

Reply:
Nope.   Consolidated.

Thanks!